lib/cartopy/io/ogc_clients.py

```diff
@@ -232,8 +232,8 @@ def _image_and_extent(self, wms_proj, wms_srs, wms_extent, output_proj,
         return LocatedImage(wms_image, extent)
 
     def fetch_raster(self, projection, extent, target_resolution):
-        service = self.service
         min_x, max_x, min_y, max_y = extent
+        target_resolution = [np.ceil(val) for val in target_resolution]
```

Comment:
To clarify: If `min_x = 19.2` then `np.ceil(19.2) == 20`. If this is a minimum value, and you make it larger, does that not make the actual extent smaller?